dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.8-slim-buster
+
+ADD requirement.txt .
+RUN python -m pip install -r requirement.txt
+# RUN export FLASK_APP="flaskr/backservice.py"
```

Comment:
1. The "flaskr" is the name of the folder. Apperntly it is a common use for a folder name in a flask apps
2. You can ignore this dockerfile since I have a didecated branch for the dockerfile and compose